--- /usr/share/plasma/plasmoids/org.kde.plasma.taskmanager/contents/ui/code/layout.js	2018-02-26 10:03:48.212666376 +0200
+++ /usr/share/plasma/plasmoids/org.kde.plasma.taskmanager/contents/ui/code/layout.js	2018-02-26 10:03:43.866062534 +0200
@@ -136,7 +136,7 @@
 function preferredMaxHeight() {
     if (tasks.vertical) {
         return verticalMargins()
-                + Math.min(
+                + Math.max(
                     // Do not allow the preferred icon size to exceed the width of the vertical task manager.
                     tasks.width,
                     Math.max(
